sdk/monitor/azure-monitor-query/samples/async_samples/sample_metrics_batch_query_async.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+"""
+FILE: sample_metrics_batch_query_async.py
+DESCRIPTION:
+    This sample demonstrates authenticating the MetricsBatchClient and retrieving the "Ingress"
+    metric along with the "Average" aggregation type for multiple resources.
+    The query will execute over a timespan of 2 hours with a granularity of 5 minutes.
+USAGE:
+    python sample_metrics_batch_query_async.py
+    2) AZURE_METRICS_ENDPOINT - The regional metrics endpoint to use (i.e. https://westus3.metrics.monitor.azure.com)
+
+    This example uses DefaultAzureCredential, which requests a token from Azure Active Directory.
+    For more information on DefaultAzureCredential, see https://learn.microsoft.com/python/api/overview/azure/identity-readme?view=azure-python#defaultazurecredential.
+
+    In this example, storage account URIs are taken.
```

Comment:
Maybe elaborate what's meant by "taken"? Ignore me if it's obvious for anyone that has knowledge of this service/library, but I wasn't sure what this meant.